docs/modules/ROOT/pages/how-tos/enable-chroot.adoc

```diff
@@ -0,0 +1,31 @@
+= Enable chrooted NGINX process
+
+== Overview
+
+Ingress NGINX v1.2.0 introduces the option to essentially isolate the web proxy server as a container inside the controller container.
+This prevents that miss-configuration of the NGINX proxy server can expose the ingress controller.
+
+For a more detailed explanation of what this does and why it increases the security of the ingress controller, you can read the https://kubernetes.io/blog/2022/04/28/ingress-nginx-1-2-0/[official blog post].
+
+== Steps
+
+[IMPORTANT]
+====
+Enabling the chrooted web proxy server adds the capability `SYS_CHROOT`.
+If you want to enable this feature on a cluster with restrictive security policies, please make sure that the NGINX deployment has permission to use the `SYS_CHROOT` capability **before** you do so.
+====
```

Comment:
Feels like the component should ensure that required configurations (e.g. PSA labels on the namespace) are present if chroot is requested.

Reply:
For PSA, you're right. However, it actually already needs the priviledged PSS as the helm chart sets `allowPrivilegeEscalation: true`. I guess we should set the label anyway, but I'd do that in a different PR.

For PSPs I suspect this will cause more issues that it solves. Not sure if that is worth the effort

In any case I would leave the note for cases where there are other admission webhooks like kyverno. We can't cover any potential security policy.

Reply:
Ok, that makes sense. I wouldn't worry about PSPs anymore at all for new features. Good point about other third-pardy admission webhooks.